Ports/composer/package.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=composer
-version="2.3.4"
```

Comment:
Please keep the quotes (or turn them into single quotes if you want) - one day I'll go on a rampage and update all ports to quote everything. It's the right thing to do, these are strings after all.